patches/local-storage.diff

```diff
@@ -52,7 +52,7 @@ Index: code-server/lib/vscode/src/vs/workbench/services/environment/browser/envi
  	get logFile(): URI { return joinPath(this.logsHome, 'window.log'); }
  
  	@memoize
--	get userRoamingDataHome(): URI { return URI.file('/User').with({ scheme: Schemas.userData }); }
+-	get userRoamingDataHome(): URI { return URI.file('/User').with({ scheme: Schemas.vscodeUserData }); }
 +	get userRoamingDataHome(): URI { return joinPath(URI.file(this.userDataPath).with({ scheme: Schemas.vscodeRemote }), 'User'); }
 +
 +	get userDataPath(): string {
```

Comment:
I was a little confused on this one. I think I did this right but there's a chance I didn't.

Reply:
Looks good!  It seems they renamed `Schemas.userData` to `Schemas.vscodeUserData` but we replace this entire line so it does not matter much to us.